tasks/interactive-bots.md

```diff
@@ -31,106 +31,126 @@ can be integrated with various APIs.
 
 ## Task Descriptions
 
-There are four types of task in this category, each corresponding to one of the
+There are four types of tasks in this category, each corresponding to one of the
  Task Types below.
 
-### Task Type A: Learn about interactive bots by running the followup bot
+### Task Type A: Learn about interactive bots by running the 'followup bot'
 
-Follow the tutorial below to test run the followup bot on your local instance.
+Follow the tutorial below to test run the followup bot on your Zulip instance
+(either local or in your droplet).
 
 * Run the Zulip server and log in the browser as Cordelia.
 
 * Find the API keys.
 
-  You can get API keys just below the bots section in the settings page. This 
+  You can get API keys just below the bots section in the settings page. This
   is how Zulip knows the request is from an authorized user.
 
   You can also create a new bot for a user in settings - it will have its own
-  api key and email. Remember to subscribe it to the streams you want to use the
+  API key and email. Remember to subscribe it to the streams you want to use the
    bot in.
 
-* Create a `~/.zuliprc-local` file with credentials:
-
+* Create a `~/.zuliprc-local` file with credentials (`nano ~/.zuliprc-local`):
   ```
   [api]
-  key=<api-key>
-  email=<email>
-  site=<dev-url>
+  key=<api-key> (Of the bot)
```

Comment:
Hm... it might be confusing and students can paste parenthesis content as well. I'd remove it.